### PR TITLE
Silent authentication

### PR DESCRIPTION
**Pull request by oskanberg**

### CODE REVIEW

**Comment by oskanberg** [comment]
It would be great to get some reviews!

File: package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-prettier": "^2.1.2",
+    "istanbul": "^0.4.5",
+    "mocha": "^3.4.2",
+    "prettier": "^1.5.3",
+    "webpack": "^3.5.2"
+  },
+  "name": "and-sso-js",
+  "description": "Client library of helpers for integration with and-gateway sso.",
+  "version": "0.0.1",
```
Comment (paul-pop) [inline package.json]: Could be 1.0.0, and would it be possible to move these to the top? name, description, version, repos, author etc.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      },
+      options
+    );
+
+    this._auth0Client = new auth0.WebAuth(this._options);
+    this._cookie = new Cookies();
+  }
+
+  // /**
+  // * Automatically (and silently) updates sso token cookie before it expires. Preempts expiry by renewOverlapSeconds seconds.
```
Comment (paul-pop) [inline src/index.js]: How come this is commented out?

File: test/unit.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const { expect } = require("chai");
+const ANDGatewayClient = require("../build/index.js");
+
+describe("handler", () => {
```
Comment (paul-pop) [inline test/unit.js]: More tests?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,46 @@
 # and-sso-js
+
```
Comment (ellieCrome) [inline README.md]: no information about how to actually build the project (yarn dev)

File: src/index.js (hunk 2 above)
Comment (oskanberg) [reply src/index.js]: Not implemented fully. Will remove commented code and put it in a separate branch.

File: test/unit.js (hunk 3 above)
Comment (oskanberg) [reply test/unit.js]: I don't know what you mean, there's plenty! /s

Comment (paul-pop) [reply test/unit.js]: Hah! Ok, will approve once we get more tests in :)

**Pull request merged**